Document/0x06b-Basic-Security-Testing.md

```diff
@@ -359,7 +372,47 @@ Possible values for the property [UIDeviceFamily](https://developer.apple.com/li
 
 The following sections describes on how to retrieve basic information of an iOS app, that might be useful during a penetration test.
 
--- ToDo Objection: https://github.com/OWASP/owasp-mstg/issues/1247
+##### Mobile Security Framework (MobSF)
+
+MobSF is a penetration testing framework that is capable of analysing IPA files and can be used before even installing the app on your testing device.
+
+Once you have MobSF up and running you can open it in your browser by navigating to <http://127.0.0.1:8000>. Just select the IPA you want to analyse and MobSF will start it's job. The bigger the app the longer it takes, but usually you should get some feedback within a few minutes.
```

Comment:
it's -> its
Just -> Simply (maybe a little bit subjective)

Reply:
done